draft-add-requirements.md

```diff
@@ -142,18 +271,97 @@ Narrative required.
+normal recursive resolvers), or be designed to serve a subset of names managed by the content provider (like an
+authoritative resolver). Using these resolvers can allow the content provider to directly control how DNS answers are
+used for load balancing and address selection, which could improve performance of connections to the content provider.
+
+Using a content-provider's encrypted resolver can also provide several privacy and security benefits: 
+
+- Prevent devices along the network path to the content-provider's resolver from observing client DNS messages
+- Verify that answers come from the entity that manages the domains being resolved
+- Reduce the number of entities able to monitor the specific names accessed by a client to only the client and the content provider, assuming that the content provider would already see the names upon a secure connection later being made based on the DNS answers (e.g., in the TLS SNI extension)
 
 | Requirement | Description |
 | R12.1 | Example requirement |
 
+# Privacy and security requirements {#priv-sec}
+
+Encrypted DNS improves the privacy and security of DNS queries and answers in the presence of malicious
+attackers. Such attackers are assumed to interfere with or otherwise impede DNS traffic and corresponding
+discovery mechanisms. They may be on-path or off-path between the client and entities with which the client
+communicates {{?RFC3552}}. These attackers can inject, tamper, or otherwise interfere with traffic as needed.
+Given these capabilities, an attacker may have a variety of goals, including, though not limited to:
+
+- Monitor and profile clients by observing unencrypted DNS traffic
+
+- Modify unencrypted DNS traffic to filter or augment the user experience
+
+- Block encrypted DNS
+
+Clients cannot assume that their network does not have such an attacker unless given some means of authenticating or otherwise
+trusting the communication with their DNS resolver.
+
+Given this type of attacker, resolver discovery mechanisms must be designed carefully to not worsen a client's security or
+privacy posture. In particular, attackers must not be able to:
+
+- Redirect secure DNS traffic to themselves when they would not otherwise handle DNS traffic.
+
+- Override or interfere with the resolver preferences of a user or administrator.
+
+- Cause clients to use a discovered resolver which has no authenticated delegation from a client-known entity.
+
+- Influence automatic discovery mechanisms such that a client uses one or more resolvers that are not
+otherwise involved with providing service to the client, such as: a network provider, a VPN server, a
+content provider being accessed, or a server that the client has manually configured.
+
+Beyond these requirements, standards describing resolver discovery mechanisms must not place any requirements
+on clients to select particular resolvers over others.
```

Comment:
```suggestion
on clients to select particular resolvers over others.

When discovering DNS resolvers on a local network, clients have no mechanism to distinguish between cases 
where an active attacker with the above capabilities is interfering with discovery, and situations wherein 
the network has no encrypted resolver. Absent such a mechanism, an attacker can always succeed in these
goals. Therefore, in such circumstances, viable solutions for local DNS resolver discovery should consider 
weaker attackers, such as those with only passive eavesdropping capabilities. It is unknown whether such
relaxations represent a realistic attacker in practice. Thus, local discovery solutions designed around
this threat model may have limited value.
```

Reply:
An attacker typically will not be always on-path and an endpoint can remember the network offers encrypted DNS server to defend against future attacks blocking the discovery.